recipe/meta.yaml

```diff
@@ -7,7 +7,7 @@ package:
   version: {{ version }}
 
 source:
-  - url: http://archive.apache.org/dist/thrift/{{ version }}/thrift-{{ version }}.tar.gz
+  - url: https://archive.apache.org/dist/thrift/{{ version }}/thrift-{{ version }}.tar.gz
     sha256: d5883566d161f8f6ddd4e21f3a9e3e6b8272799d054820f1c25b11e86718f86b
     patches:
       - thrift-0.13.0-windows-fixes.patch  # [win]
```

Comment:
should there be sha's for the other zips?

Reply:
Done.